docs/plugins-database.md

```diff
@@ -0,0 +1,386 @@
+---
```

Comment:
Rename back to `plugins.md`. I changed the name in #7804 to avoid deleting the original plugins list,but that's no longer a concern here.